README.md

```diff
@@ -26,15 +26,15 @@ c = a;
 ### Your solution here:
 1.  What is `a`?
 ```
-a is ...
+a is ... number
```

Comment:
The required here is the value.
`a = 1` then ` a = b` and `b = 'bongos'`
so `a` be `'bongos'`
```suggestion
a is ... 'bongos'
```